briefcase/app.py

```diff
@@ -70,6 +78,10 @@ def initialize_options(self):
         self.version_code = None
         self.guid = None
         self.secret_key = None
+        self.build = False
+        self.execute = False
+        self.os_version = 'iOS 10.2'
```

Comment:
Related to the above, let's set these props to `None` or something in the general `app.py`, and set the specific values in ios.py